src/TiledArray/math/linalg/cholesky.h

```diff
@@ -75,10 +75,10 @@ auto cholesky_lsolve(TransposeFlag transpose, const Array& A, const Array& B,
 }  // namespace TiledArray::math::linalg
 
 namespace TiledArray {
-  using TiledArray::math::linalg::cholesky;
-  using TiledArray::math::linalg::cholesky_linv;
-  using TiledArray::math::linalg::cholesky_solve;
-  using TiledArray::math::linalg::cholesky_lsolve;
-}
+using TiledArray::math::linalg::cholesky;
```

Comment:
Don't these aliases defeat the purpose of nesting the `math::linalg` entirely?

Reply:
@asadchev was in charge of laying out namespaces, I don't recall the rationale; @asadchev please comment